packages/jest-haste-map/src/index.js

```diff
@@ -325,6 +325,28 @@ class HasteMap extends EventEmitter {
           throw new Error(message);
         }
         this._console.warn(message);
```

Comment:
Should we remove this warning (Can be done in a follow-up PR)? It doesn't provide much use.